client/apps/settings/client/curations.coffee

```diff
@@ -1,10 +1,31 @@
-AdminEditView = require '../../../components/admin_form/index.coffee'
+Backbone = require('backbone')
+{ AdminEditView } = require '../../../components/admin_form/index.coffee'
 Curation = require '../../../models/curation.coffee'
 sd = require('sharify').data
-_s = require 'underscore.string'
+_ = require('underscore')
 
-@init = ->
-  new AdminEditView
-    model: new Curation(sd.CURATION)
-    el: $('body')
-    onDeleteUrl: '/settings/curations'
+
+module.exports.CurationEditView = class CurationEditView extends Backbone.View
+
+  events:
+    'click .edit-feature__section-title': 'revealSection'
+
+  initialize: ->
+    @curation = new Curation sd.CURATION
+    new AdminEditView
+      model: @curation
+      el: $('body')
+      onDeleteUrl: '/settings/curations'
+    if this.curation.get('type') == "editorial-feature"
```

Comment:
Minor coffeescript code ⛳️  tip: `@initMenuState() if @curation.get('type') is 'editorial-feature'`